pkg/operator/ceph/config/monstore.go

```diff
@@ -93,7 +93,7 @@ func (m *MonStore) SetIfChanged(who, option, value string) (bool, error) {
 // Set sets a config in the centralized mon configuration database.
 // https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
 func (m *MonStore) Set(who, option, value string) error {
-	logger.Infof("setting %q=%q=%q option to the mon configuration database", who, option, value)
+	logger.Infof("setting %q=%q option to the mon configuration database", who, option)
```

Comment:
How about if we make this log entry a bit more clear? It's a bit confusing that the log would show `user=key`:
```suggestion
	logger.Infof("setting option %q (user %q) to the mon configuration database", option, who)
```